frontend/package.json

```diff
@@ -62,11 +62,11 @@
     ]
   },
   "dependencies": {
-    "@patternfly/patternfly": "2.8.2",
+    "@patternfly/patternfly": "2.13.0",
```

Comment:
#1759 has some of the same bumps. We should probably use 2.13.1 here

Reply:
just wanted to see if CI would be happy first haha ;) - will bump it again shortly